iotedgedev/modules.py

```diff
@@ -163,3 +168,31 @@ def deploy_device_configuration(
             self.output.error(
                 "There was an error deploying the configuration. Please make sure your IOTHUB_CONNECTION_STRING and DEVICE_CONNECTION_STRING Environment Variables are correct.")
 
+class ModulesProcessorFactory(Modules):
+    
+    def factory(self, module_language):
+        if module_language == "csharp" or  module_language == "fsharp" or module_language == "vbasic":
```

Comment:
If you do a compare like this you want to make sure they are the same casing.  I know you called to Lower before, but this class shouldn't assume that is done.